package.json

```diff
@@ -70,7 +70,7 @@
     "postpack": "shx rm -f oclif.manifest.json",
     "posttest": "yarn lint",
     "prepack": "yarn build && oclif manifest && oclif readme",
-    "test": "mocha --forbid-only \"test/**/*.test.ts\"",
+    "test": "mocha --forbid-only \"test/**/nodejs.test.ts\"",
```

Comment:
I'm assuming this is intended to be temporary?

Reply:
you can run `yarn test -g nodejs`

Reply:
nice I didn't know that